### PR TITLE
test(adminapi): expand coverage for trust marks, stats, and subordinate flows

### PR DESCRIPTION
**Pull request by hmffa**
## Summary
This PR expands admin API test coverage across additional handler surfaces and edge cases, with a focus on trust mark management, stats endpoints, subordinate behavior, and storage-backed persistence flows.

Highlights:
- add tests for entity trust marks CRUD and real storage persistence
- add coverage for stats summary, top breakdowns, time series, latency/daily views, and export endpoints
- add tests for subordinate event helpers, duplicate claims and critical extensions, constraint edge cases, and statement lifetime default/error fallback behavior
- strengthen trust mark type coverage for duplicate and not found cases
- add real storage persistence tests for trust mark issuance specs and subjects
- refactor existing user and trust mark tests to use more consistent request handling, error assertions, and non-parallel execution where state is shared

### CODE REVIEW

File: api/adminapi/test_helpers_test.go [hunk 1]
```diff
@@ -57,6 +58,26 @@ func assertStatusOneOf(t *testing.T, resp *http.Response, expected ...int) {
...
 }
 
+// assertErrorResponse checks the HTTP status code and verifies the JSON error body
+// contains the expected "error" field (e.g., "invalid_request", "not_found", "server_error").
+// Use this for error path tests instead of assertStatus when you want to verify the full API contract.
+func assertErrorResponse(t *testing.T, resp *http.Response, body []byte, expectedStatus int, expectedError string) {
+	t.Helper()
+	assertStatus(t, resp, expectedStatus)
+
+	var errBody struct {
```
[the remark this answers is not in the document]
Comment (hmffa) [reply api/adminapi/test_helpers_test.go:64]: Refactored the tests to do body checking also
